docs/source/schema-design/federated-schemas/entities/intro.mdx

```diff
@@ -1,24 +1,24 @@
 ---
 title: Introduction to Entities
-subtitle: Resolve federated types across multiple subgraphs
-description: Learn to define, contribute to, and reference object types that resolve their fields across multiple subgraphs in a federated GraphQL architecture.
+subtitle: Fundamental keyed object type of Apollo Federation
+description: Learn to define, contribute to, and reference entities, the fundamental object types of Apollo Federation that resolve their fields across one or more subgraphs.
 redirectFrom:
     - /federation/entities
 ---
 
-Federated schemas let multiple subgraphs collaboratively define and resolve fields for shared object types.
-This guide shows you how to define a shared object type called an _entity_.
+<Note>
 
 Before you get started, you may want to check out the [Introduction to Apollo Federation](/graphos/schema-design/federated-schemas/federation) for a conceptual overview.
 
+</Note>
+
 ## Entity overview
 
-_Entities_ are a fundamental aspect of federated schemas.
-In a supergraph, an entity is an object type that can resolve its fields across multiple subgraphs.
-Each subgraph can contribute different fields to the entity and is responsible for resolving only the fields that it contributes.
-This enables subgraphs to adhere to the separation of concerns principle.
+In Apollo Federation, federated data objects are represented as _entities_. Entities are objects that can be fetched with primary keys. Like a row in a database table, an entity contains fields of various types, and it can be uniquely identified by a key field or set of fields.
```

Comment:
I'm torn about using "unique identifier" vs. "primary key" here. The former is more technically accurate while the latter helps set up the next sentences. I think my preference is for the former--wdyt?

```suggestion
In Apollo Federation, federated data objects are represented as _entities_. Entities are objects that can be fetched with unique identifiers. Like a row in a database table, an entity contains fields of various types, and it can be uniquely identified by a key field or set of fields.
```

Reply:
Is it because entity keys are mostly but not completely same as a databse primary keys?

I think introducing "key" in the first definition is (ahem) key 😁 for entities.

how about: 

   ```suggestion
In Apollo Federation, federated data objects are represented as _entities_. Entities are objects that can be fetched with one or more unique key fields. Like a row in a database table, an entity contains fields of various types, and it can be uniquely identified by a key field or set of fields.
```

Reply:
WFM!